### PR TITLE
[zh-cn] sync some zh-cn doc

### PR DESCRIPTION
**Pull request by bzsuni**
Modifications include the following：
 content/zh-cn/docs/concepts/policy/resource-quotas.md
 content/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md
 content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md
 content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md
 content/zh-cn/docs/concepts/services-networking/ingress-controllers.md

### CODE REVIEW

**Review by windsonsea — APPROVED** [review]
One suggestion to you:

This kind of change is very suitable for a newcomer.

But if you [spam many PRs related to such a subtle change](https://github.com/kubernetes/website/pulls?q=is%3Apr+author%3Abzsuni+is%3Aclosed), that will be not an encouraged behavior and may cause a negative effect to your impression herein.

**Comment by bzsuni** [comment]
This pr is not ready, after that there are still some changes that need to be made

File: content/zh-cn/docs/concepts/services-networking/ingress-controllers.md [hunk 1]
```diff
@@ -134,6 +135,9 @@ Kubernetes 作为一个项目，目前支持和维护
   网页服务器（作为代理）一起使用。
 * [ngrok Kubernetes Ingress 控制器](https://github.com/ngrok/kubernetes-ingress-controller)
   是一个开源控制器，通过使用 [ngrok 平台](https://ngrok.com)为你的 K8s 服务添加安全的公开访问权限。
+* [OCI Native Ingress 控制器](https://github.com/oracle/oci-native-ingress-controller#readme)
+  是适用于 Oracle Cloud Infrastructure 的 Ingress 控制器，允许您管理
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/ingress-controllers.md:139]:
```suggestion
  是适用于 Oracle Cloud Infrastructure 的 Ingress 控制器，允许你管理
```

**Comment by windsonsea** [comment]
This can be closed. Something new changes are introduced from the en upstream. Thanks for your contribution. Feel free to re-open this kind PR if you are available.

/close

**Pull request closed without merging**